### PR TITLE
Fix Issue #347: Remove wording that implies ballot chaining

### PR DESCRIPTION
**Pull request by No-Cellist-7780**
_🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository._

### Issue
Fixes issue #347 

### Description
Renamed `get_rotating_ballot_code` to `get_ballot_code` and `previous_code` to `code_seed`

### CODE REVIEW

**Comment by No-Cellist-7780** [comment]
PTAL

File: src/electionguard/ballot.py [hunk 1]
```diff
@@ -864,7 +864,7 @@ def make_ciphertext_ballot(
     :param crypto_base_hash: Hash of the cryptographic election context
     :param contests: List of contests for this ballot
     :param timestamp: Timestamp at which the ballot encryption is generated in tick
-    :param previous_code: Previous ballot code or seed
+    :param code_seed: Previous ballot code or seed
```
Comment (keithrfung) [inline src/electionguard/ballot.py:867]: The descriptions should be changed as well. This one should be `Seed for ballot code`

File: src/electionguard/ballot.py [hunk 2]
```diff
@@ -877,9 +877,7 @@ def make_ciphertext_ballot(
     if previous_ballot_code is None:
```
Comment (keithrfung) [inline src/electionguard/ballot.py:877]: This linen should also be `code_seed`.

 I would do a search for `previous_ballot_code`

**Review by keithrfung — CHANGES_REQUESTED** [review]
Wow. Great work! Only a couple of changes which should be pretty light.

**Comment by keithrfung** [comment]
Apologies for the late review. Been one of those weeks

File: src/electionguard/ballot.py (hunk 1 above)
Comment (No-Cellist-7780) [reply src/electionguard/ballot.py:867]: alright

File: src/electionguard/ballot.py (hunk 2 above)
Comment (No-Cellist-7780) [reply src/electionguard/ballot.py]: sorry overlooked those

**Review by keithrfung — APPROVED** [review]

**Pull request merged**